pkg/operation/common/utils.go

```diff
@@ -444,6 +444,86 @@ func ShouldObjectBeRemoved(obj metav1.Object, gracePeriod time.Duration) bool {
 	return deletionTimestamp.Time.Before(time.Now().Add(-gracePeriod))
 }
 
+// DeleteVpa delete all resources required for the vertical pod autoscaler in the given namespace.
+func DeleteVpa(k8sClient kubernetes.Interface, namespace string) error {
+	if k8sClient == nil {
+		return fmt.Errorf("require kubernetes client")
+	}
+
+	listOptions := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("%s=%s", GardenRole, GardenRoleVpa),
+	}
+
+	// Delete all Crds with label "garden.sapcloud.io/role=vpa"
+	if err := k8sClient.APIExtension().Apiextensions().CustomResourceDefinitions().DeleteCollection(
+		&metav1.DeleteOptions{
+			TypeMeta: metav1.TypeMeta{
```

Comment:
Why is this needed?

Reply:
I'm not sure I understand. Do you mean why do I need delete options or why am I deleting the crds?

Reply:
Why do you need the options. You already specify the API version and kind, so why is it needed twice?

Reply:
I didn't know it worked without the options. I'll change it to pass an empty DeleteOption struct.

Reply:
I don't know whether it works without the options. I just wanted you to clarify why it was implemented that way.